src/components/structures/LoggedInView.js

```diff
@@ -322,6 +322,14 @@ const LoggedInView = React.createClass({
                     handled = true;
                 }
                 break;
+            case KeyCode.KEY_I:
```

Comment:
Any particular meaning to choosing the letter?

Reply:
Was the first non-conflict key that made some amount of sense. I for "information" or something.